ui/src/components/view/DetailSettings.vue

```diff
@@ -85,6 +85,7 @@
           <tooltip-button
             :tooltip="$t('label.edit')"
             icon="edit"
+            :disabled="deployasistemplate === true && (item.name === 'rootDiskController' || item.name === 'dataDiskController')"
```

Comment:
I think `deployasistemplate === true` should be enough in this condition

Reply:
Done